client/src/pages/private/ParticipantDetailsView.js

```diff
@@ -170,16 +169,18 @@ export default () => {
   };
 
   // Navigate on link
-  const handleNavigateBackLink = () => {
+  const getNavigateBackLink = () => {
     switch (linkName) {
       case 'Participant':
-        history.push(Routes.ParticipantView);
-        break;
+        return Routes.ParticipantView;
       case 'Site View':
-        history.push(Routes.SiteView + `/${pageId}`);
-        break;
+        return Routes.SiteView + `/${pageId}`;
+      case 'Cohort':
+        return keyedString(Routes.CohortDetails, {
+          id: pageId,
+        });
       default:
-        history.goBack();
+        return Routes.ParticipantView;
```

Comment:
I don't know of any cases where we haven't arrived here from pages not listed above, but as a fallback we'll send them back to the main page